lib/error.js

```diff
@@ -79,9 +79,12 @@ const serializeMessageItem = messageItem => {
 	return '';
 };
 
+// Indicates that the error is used only to interrupt control flow, but not in the return value
+export class DiscardedError extends Error {}
```

Comment:
For example, when a process ends with `SIGTERM`, we throw an error to interrupt the flow. But we haven't constructed the error and its message yet. So we use this dummy error class.